File: 0724-find-pivot-index/0724-find-pivot-index.cs
public class Solution {
    public int PivotIndex(int[] nums) {
        int sum = nums.Sum();
        int leftSum = 0;
        for(int i = 0; i < nums.Length; i++){
            if (
                (sum - nums[i])== 2*leftSum)
                return i;

            leftSum += nums[i];
        }

        return -1;
    }
}

File: 0724-find-pivot-index/0724-find-pivot-index.cpp
class Solution {
public:
    int pivotIndex(vector<int>& nums) {
       int n=nums.size();
        int s=0;
        for(int i: nums)s+=i;
        int l=0;
        for(int i=0;i<n;i++)
        {
           
            if(2*l==s-nums[i])return i;
            l+=nums[i];
        }
        return -1;
    }
};